pkg/workload/kv/kv.go

```diff
@@ -942,40 +965,90 @@ func (g *sequentialGenerator) state() string {
 	return fmt.Sprintf("S%d", g.seq.read())
 }
 
-type zipfGenerator struct {
-	seq    *sequence
+type writeSequenceSource struct {
+	*sequence
+}
+
+func (wss writeSequenceSource) Int63() int64 {
+	return int64(wss.Uint64() & 0x7fffffffffffffff) // Masks high bit
+}
+
+func (wss writeSequenceSource) Uint64() uint64 {
+	hasher := crc64.New(crc64.MakeTable(crc64.ECMA))
```

Comment:
`MakeTable` can get expensive on _every_ put. Why not reuse?

Reply:
`MakeTable()` caches the table creation and returns a value behind a `sync.Once`, so this call is actually cheap.

Reply:
🤦 I see, looked at the internal `makeTable`. In that case, you're still paying the price of alloc, but it looks like it should fit into the stack.